src/wp-includes/blocks.php

```diff
@@ -153,7 +153,8 @@ function register_block_script_handle( $metadata, $field_name, $index = 0 ) {
 		$script_handle,
 		$script_uri,
 		$script_dependencies,
-		isset( $script_asset['version'] ) ? $script_asset['version'] : false
+		isset( $script_asset['version'] ) ? $script_asset['version'] : false,
+		array( 'in_footer' => false )
```

Comment:
This change seems unnecessary? This is the default value, so no need to provide this parameter here I think?